tech.json

```diff
@@ -345,21 +346,21 @@
             "Gravity"
           ],
           "note": [
-            "Turning off Gravity Suit right after the start of an underwater jump to achieve higher height.",
+            "Turning off Gravity Suit right after the start of an underwater jump to achieve a much higher jump.",
             "Can be performed in water, lava, or acid."
           ]
         },
         {
           "name": "canIframeSpikeJump",
           "requires": [ "canCarefulJump" ],
-          "note": "Utilizing spikes i-frames to run and jump on spike floors or enemy i-frames to setup a jump from a spike wall."
+          "note": "Take damage to gain invincibility frames so that Samus can to run and jump on spike floors or setup a wall jump from a spike wall.  Enemies provide more i-frames than spikes."
```

Comment:
```suggestion
          "note": "Take damage to gain invincibility frames so that Samus can run and jump on spike floors or setup a wall jump from a spike wall.  Enemies provide more i-frames than spikes."
```